Unaccountable-Magpie/BangazonSite
# Logged in users can click sell something

### Purpose: 
To enforce that only users who have logged in can sell a product


### How it fits in context of the project:
Fulfills [this ticket](https://github.com/Unaccountable-Magpie/BangazonSite/issues/33)


### Specific Feature affected:
The nav bar should have a conditionally rendered button that says "Sell Something". When the signed in user clicks it, they should be presented with a form to sell items.


### How to test (Be thorough!):

1. checkout my branch ag-sellproduct
2. start bangazon.sln
3. click the green express button to run the code
4. the nav bar should not show "Sell Something"
5. register and log in
6. the nav bar should now contain a link "Sell Something"
7. click the link and you should see a form that will eventually work but does not at this time.
8. To run unit testing, type dotnet test into the nuget package manager and ensure that the 3 tests pass.
